packages/core/src/constants.ts

```diff
@@ -19,6 +19,8 @@ export const LATITUDE_SLACK_URL =
   'https://trylatitude.slack.com/join/shared_invite/zt-17dyj4elt-rwM~h2OorAA3NtgmibhnLA#/shared-invite/email'
 export const LATITUDE_HELP_URL = LATITUDE_SLACK_URL
 export const HEAD_COMMIT = 'live'
+export const MAX_FREE_RUNS = 2000
+
```

Comment:
Since there is no concept of plan yet, I'm just hardcoding this value. Database schema is the exact same for now.